source/docs/javascript_usage.md

```diff
@@ -4,6 +4,26 @@ title: Usage in Javascript
 Embark includes in your dapp the EmbarkJS library. This library abstracts different functionality so you can easily & quickly build powerful dapps that leverage different decentralized technologies.
 Embark will automatically initialize EmbarkJS with the configurations set for your particular environment.
 
+### Provider
+
+According to [EIP1102](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1102.md), Dapps MUST request account access and Embark allows you to choose how you want to deal with request access.
+
+In the smart contracts configuration you can find this snippets:
+<pre><code class="javascript">
+...
+  // Automatically call `ethereum.enable` if true.
+  // If false, the following code must run before sending any transaction: `await EmbarkJS.enableEthereum();`
+  // Default value is true.
+  // dappAutoEnable: true,
+...
+</code></pre>
+
+By default the value of `dappAutoEnable` is true which means that Embark will call for you `ethereum.enable` to request account access when the first page of the dapp is loaded.
+
+If you want more control you can set `dappAutoEnable` to false.
+Then if you want to request account, you can use the following function: `await EmbarkJS.enableEthereum();`.
+This will request account access and once granted you will be able to make transaction call.
```

Comment:
As mentioned above, it seems that `web3.enable()` is already deprecated as per spec. So we might want to propagate whatever is being emitted by `enableEthereum()` (in this case it would be the requested accounts).